flashinfer/gemm/routergemm.py

```
@@ -305,7 +305,22 @@ def tinygemm2_op_impl(
     ) -> None:
         module.tinygemm2_op(input, weight, bias, out, use_pdl)
 
-    return SimpleNamespace(tinygemm2_op=tinygemm2_op_impl)
+    @register_custom_op(
+        "flashinfer::tinygemm2_nobias_op",
+        mutates_args=["out"],
+    )
+    def tinygemm2_nobias_op_impl(
+        input: torch.Tensor,
```

Ruff (0.15.10)

[error] Function argument `input` is shadowing a Python builtin

(A002)